cluster/config-defaults.yaml

```diff
@@ -86,7 +86,7 @@ skipper_suppress_route_update_logs: "true"
 skipper_validate_query: "true"
 skipper_validate_query_log: "false"
 
-skipper_default_filters: 'enableAccessLog(4,5) -> fifo(2000,20000,"3s")'
+skipper_default_filters: 'disableAccessLog(2,3,404,429) -> fifo(2000,20000,"3s")'
```

Comment:
Do we log 1xx by this config?

Reply:
yes but 1xx is not very often and would be even interesting to see IMO.